config/jobs/GoogleCloudPlatform/k8s-cluster-bundle/k8s-cluster-bundle.yaml

```diff
@@ -27,3 +33,9 @@ postsubmits:
         args: 
         - "test"
         - "..."
+        volumeMounts:
```

Comment:
make a preset?

Reply:
nah... will remove after upstream fix get in